### PR TITLE
Auto Focus for Phone Number Input

### PR DESCRIPTION
**Pull request by akbarjorayev**

### CODE REVIEW

**Review by akbarjorayev — COMMENTED** [review]
Good

**Review by akbarjorayev — COMMENTED** [review]
**Code** 🔥

File: src/pages/Login.js [hunk 1]
```diff
@@ -23,8 +23,24 @@ function Login() {
...
+    const sections = [2, 6, 9];
+    const separators = [' ', '-', '-'];
+
+    let i = 0;
+    for (const length of sections) {
+      if (formattedPhoneNumber.length > length) {
+        formattedPhoneNumber =
+          formattedPhoneNumber.substring(0, length) +
+          separators[i] +
+          formattedPhoneNumber.substring(length);
```
Comment (nazarovctrl) [inline src/pages/Login.js:38]:
```suggestion
        formattedPhoneNumber = formattedPhoneNumber.substring(0, length) + separators[i] + formattedPhoneNumber.substring(length);
```

**Review by nazarovctrl — CHANGES_REQUESTED** [review]

File: src/pages/Login.js (hunk 1 above)
Comment (akbarjorayev) [reply src/pages/Login.js:38]: Nima ozgargan?

Comment (nazarovctrl) [reply src/pages/Login.js:38]: image
 1ta qator bogan

File: src/pages/Login.js [hunk 2]
```diff
@@ -170,19 +183,26 @@ function Login() {
               <br></br>
               <form action="" onSubmit={handleSubmit} style={{ width: '100%' }}>
                 <div className="phone-container">
-                  <span className="country-code">+998</span>
-                  <InputMask
```
Comment (nazarovctrl) [inline src/pages/Login.js:174]: InputMask import tepada qob ketgan, lekin InputMask   Login.js da ishlatilmayapti , shunga importini ob tashash kere

**Review by nazarovctrl — CHANGES_REQUESTED** [review]

**Review by akbarjorayev — COMMENTED** [review]
Good

**Review by nazarovctrl — APPROVED** [review]

**Pull request merged**